metricbeat/vendor/vendor.json

```diff
@@ -2,12 +2,31 @@
 	"comment": "",
 	"ignore": "test github.com/elastic/beats",
 	"package": [
+		{
+			"checksumSHA1": "GYschMPw/Zi5C4QnGKRfgljnx/c=",
+			"path": "github.com/gogo/protobuf/proto",
```

Comment:
@exekias Can you check if some of these we also have in the global vendor directory?  If yes, we should bring the versions in sync.

Reply:
It seems we have `github.com/golang/protobuf` under base `vendor`, which differs from this one

Reply:
Should we sync them up?